db/migrate/20210413184451_create_cards.rb

```diff
@@ -0,0 +1,17 @@
+class CreateCards < ActiveRecord::Migration[6.1]
+  def change
+    create_table :cards do |t|
+      t.string :name
+      t.string :type
```

Comment:
I think this column called "type" might be an issue. Did you work through this in your test app?

Reply:
Yes, this has been renamed to "pokemon_type"